Dockerfile.slim

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.11
+FROM python:3.7-alpine3.16
```

Comment:
The changes here are the only diffs from https://github.com/spinnaker/halyard/pull/2171